src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectTypeDescriptionProvider.ReflectedTypeData.cs

```diff
@@ -45,7 +45,7 @@ internal ReflectedTypeData(Type type, bool isRegisteredType)
             /// Retrieves custom attributes.
             /// </summary>
             [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2062:UnrecognizedReflectionPattern",
-                Justification = "_type is annotated as preserve All members, so any Types returned from GetInterfaces should be preserved as well once https://github.com/mono/linker/issues/1731 is fixed.")]
```

Comment:
The justification was incorrect, the warning is on lines working with elements of `Type[]` returned by `TrimSafeReflectionHelper.GetInterfaces`, not `Type.GetInterfaces`. That will not work without warnings.